client/src/components/SelectTabs/EventTypesTab.js

```diff
@@ -25,521 +25,470 @@ import Checkmark from "../../assets/check.png";
-                                    direction="row"
-                                    alignItems="baseline"
-                                    justify="flex-start"
-                                    spacing="2"
-                                    wrap="nowrap"
-                                    className={classes.groupedInput}
-                                >
-                                    {/* TODO: pass in user link prefix */}
-                                    <Grid
-                                        xs="5"
-                                        className={classes.prefix}
-                                        item
-                                    >
-                                        calendapp.com/john-doe/
-                                    </Grid>
-                                    <Grid item>
-                                        <Input
-                                            label="Link"
-                                            name="link"
-                                            value={eventBody.link}
-                                            type="text"
-                                            onChange={handleFormChange}
-                                            disableUnderline
-                                            fullWidth
-                                        />
-                                    </Grid>
-                                </Grid>
-                            </Grid>
-                        </Grid>
-                        <Grid
-                            direction="row"
-                            alignItems="center"
-                            container
-                            justify="flex-start"
-                            className={classes.colorRow}
-                        >
-                            <Grid xs="2" item>
-                                <InputLabel className={classes.label}>
-                                    Color
-                                </InputLabel>
-                            </Grid>
+              <Grid xs="10" item>
+                <Grid
+                  container
+                  direction="row"
+                  alignItems="baseline"
+                  justify="flex-start"
+                  spacing="2"
+                  wrap="nowrap"
+                  className={classes.groupedInput}
+                >
+                  {/* TODO: pass in user link prefix */}
+                  <Grid xs="5" className={classes.prefix} item>
+                    calendapp.com/john-doe/
+                  </Grid>
+                  <Grid item>
+                    <Input
+                      label="Link"
+                      name="link"
+                      value={eventBody.link}
+                      type="text"
+                      onChange={handleFormChange}
+                      disableUnderline
+                      fullWidth
+                    />
```

Comment:
[TextField](https://material-ui.com/components/text-fields) is the recommended MUI component here